spanner/api/Spanner.Samples/DeleteUsingDmlReturningAsyncPostgre.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START spanner_postgresql_delete_dml_returning]
+
+using Google.Cloud.Spanner.Data;
+using Google.Cloud.Spanner.V1;
+using System;
+using System.Threading.Tasks;
+
+public class DeleteUsingDmlReturningAsyncPostgreSample
+{
+    public async Task<int> DeleteUsingDmlReturningAsyncPostgre(string projectId, string instanceId, string databaseId)
+    {
+        string connectionString = $"Data Source=projects/{projectId}/instances/{instanceId}/databases/{databaseId}";
+
+        using var connection = new SpannerConnection(connectionString);
+        await connection.OpenAsync();
+
+        //Deletes records from Singers table satisfying a particular condition and returns the SingerId
+        //FirstName and LastName column of the deleted records using ‘RETURNING SingerId, FirstName, LastName’.
+        //All columns of all the deleted records can be returned using ‘RETURNING *’.
+        using var cmd = connection.CreateDmlCommand("DELETE FROM Singers WHERE SingerId = 12 RETURNING *");
+
+        var reader = await cmd.ExecuteReaderAsync();
+        var rowCount = 0;
+        while (await reader.ReadAsync())
+        {
+            // See https://www.postgresql.org/docs/current/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS
```

Comment:
We don't need this comment.

Also, please address the comments in other samples here.